linux_os/guide/system/software/gnome/gnome_screen_locking/dconf_gnome_screensaver_mode_blank/rule.yml

```
@@ -26,13 +49,21 @@ severity: medium
 identifiers:
     cce@rhel7: CCE-80113-4
     cce@rhel8: CCE-80778-4
+    cce@sle12: CCE-83221-2
+    cce@sle15: CCE-85715-1
 
 references:
     cjis: 5.5.5
     cui: 3.1.10
     disa: CCI-000060
     nist: AC-11(1),CM-6(a)
     nist-csf: PR.AC-7
+    nist@sle15: AC-11(1),AC-11(1).1
```

Please condense these two nist@ into one e.g.
```suggestion
    nist: AC-11(1),AC-11(1).1
```

---

Done